### ISSUE TITLE
Crash when attempting to view object with referenced object

### ISSUE THREAD

**Issue by Danm72** [bearjaw/RBSRealmBrowser#1]
I have a 1 to 1 relationship between two objects, when i attempt to view the Person class RBS fails.

```
class Dog: Object {
    // ... other property declarations
    dynamic var owner: Person? // to-one relationships must be optional
}

let jim = Person()
let rex = Dog()
rex.owner = jim
```

fails on line 158 RBSRealmObjectsBrowser  let data = object[property.name] as! NSData

**Comment by bearjaw** [comment]
Hey,
I will try to fix it tonight. Thanks for reporting the bug.

Cheers

**Comment by bearjaw** [comment]
This should be fixed in 0.1.9. 
Try it out and get back to me and I'll close the issue.

**Comment by Danm72** [comment]
You nailed it. Nice one! Thanks a lot.

Would you recommend people use 
`pod 'RBSRealmBrowser', :git => 'https://github.com/bearjaw/RBSRealmBrowser.git'`

The podspec seems to be quite out of date.
